apps/language_models/scripts/sharded_vicuna_fp32.py

```diff
@@ -621,39 +620,36 @@ def get_sharded_model():
         )
         prompt = prompt_history.strip()
         input_ids = tokenizer(prompt).input_ids
+        tokens = input_ids
         prompt = print("Robot:", end=" ")
-        new_sentence = ""
-        for _ in range(200):
+        new_sentence = []
+        for iteration in range(200):
```

Comment:
NIT: Can we set this as a global-ish variable/constant i.e `MAX_LEN_TOKEN` for better readability. :)

Reply:
you mean 200?  would it be better to change it to while True?